.github/CODEOWNERS

```diff
@@ -11,6 +11,10 @@
 # Canvas
 /x-pack/plugins/canvas/  @elastic/kibana-canvas
 
+# Code
+/x-pack/plugins/code/  @teams/code
+/x-pack/test/functional/apps/code/  @teams/code
```

Comment:
`/x-pack/test/api_integration/apis/code` let's add this folder too.

Reply:
done